### PR TITLE
Add initial IoTea page

### PR DESCRIPTION
**Pull request by runewake2**

### CODE REVIEW

**Comment by runewake2** [comment]
Should be ok to review now.

**Comment by jimbobbennett** [comment]
The image isn't working so well in dark mode:

image

File: iot-analytics/IoTea/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+![](../../img/iot/iotea/iotea-title.svg)
```
Comment (jimbobbennett) [inline iot-analytics/IoTea/README.md:1]: Can you add alt text please

File: iot-analytics/IoTea/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# IoTea: Brewing an Azure flavored Home Supply Chain
+
+Large IoT applications can take many forms including smart buildings, digital agriculture or even smart cities and implementing them can be daunting. Join Jim and Sam as they demystify some of this by overengineering a ‘simple’ home IoT project in these live coding sessions.
```
Comment (jimbobbennett) [inline iot-analytics/IoTea/README.md:5]: Maybe a comma after smart cities.

File: iot-analytics/IoTea/README.md (hunk 1 above)
Comment (jimbobbennett) [inline iot-analytics/IoTea/README.md:1]: Can you add alt text please

File: img/iot/iotea/iotea-title.svg [hunk 3]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (jimbobbennett) [inline img/iot/iotea/iotea-title.svg:1]: The image isn't working so well in dark mode:

image

**Review by jimbobbennett — CHANGES_REQUESTED** [review]

File: img/iot/iotea/iotea-title.svg (hunk 3 above)
Comment (runewake2) [reply img/iot/iotea/iotea-title.svg:1]: Good point, I made it orange instead (maybe something blue would work as well?) The shaded version should work well on most grey backgrounds.

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**